.github/workflows/delete-storybook-vrt-report.yml

```
@@ -0,0 +1,59 @@
+name: Delete Storybook VRT Report
+
+on:
+  delete:
+    branches-ignore: [main, develop, gh-pages]
```

actionlint

"branches-ignore" filter is not available for delete event. it is only for push, pull_request, pull_request_target, workflow_run events

(events)